tests/kuttl/tests/designate_tls/05-assert.yaml

```diff
@@ -0,0 +1,246 @@
+          subPath: tls-ca-bundle.pem
+        - mountPath: /var/lib/config-data/tls/certs/internal.crt
+          name: internal-tls-certs
+          readOnly: true
+          subPath: tls.crt
+        - mountPath: /var/lib/config-data/tls/private/internal.key
+          name: internal-tls-certs
+          readOnly: true
+          subPath: tls.key
+        - mountPath: /var/lib/config-data/tls/certs/public.crt
+          name: public-tls-certs
+          readOnly: true
+          subPath: tls.crt
+        - mountPath: /var/lib/config-data/tls/private/public.key
+          name: public-tls-certs
+          readOnly: true
+          subPath: tls.key
+      initContainers:
+      - args:
+        - -c
+        - /usr/local/bin/container-scripts/init.sh
+        command:
+        - /bin/bash
+        imagePullPolicy: IfNotPresent
+        name: init
+        resources: {}
+        volumeMounts:
+        - mountPath: /usr/local/bin/container-scripts
+          name: scripts
+          readOnly: true
+        - mountPath: /var/lib/config-data/default
+          name: config-data
+          readOnly: true
+        - mountPath: /var/lib/config-data/merged
+          name: config-data-merged
+      restartPolicy: Always
+      serviceAccount: designate-designate
+      serviceAccountName: designate-designate
+      volumes:
+      - name: scripts
+        secret:
+          secretName: designate-api-scripts
+          defaultMode: 493
+      - name: config-data
+        secret:
+          secretName: designate-api-config-data
+          defaultMode: 416
+      - emptyDir: {}
+        name: config-data-merged
+      - emptyDir: {}
+        name: designate-run
+      - name: combined-ca-bundle
+        secret:
+          secretName: combined-ca-bundle
+          defaultMode: 292
+      - name: internal-tls-certs
+        secret:
+          secretName: cert-designate-internal-svc
+          defaultMode: 256
+      - name: public-tls-certs
+        secret:
+          secretName: cert-designate-public-svc
+          defaultMode: 256
+---
+# the openshift annotations can't be checked through the deployment above
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    openshift.io/scc: anyuid
+  labels:
+    service: designate
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    endpoint: internal
+    service: designate
+  name: designate-internal
+spec:
+  ports:
+    - name: designate-internal
+  selector:
+    service: designate
+  type: ClusterIP
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    endpoint: public
+    service: designate
+  name: designate-public
+spec:
+  ports:
+    - name: designate-public
+  selector:
+    service: designate
+  type: ClusterIP
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+  - script: |
+      if oc get -n $NAMESPACE secret -o json designate-central-config-data | jq '.data["designate.conf"] | @base64d' | sed -e 's/\\n/\n/g' | grep ^backend_url.*redis.*ssl.1;
```

Comment:
I don't think that I understand, on tests/kuttl/tests/basic/02-assert.yaml we have
if ! oc get -n $NAMESPACE secret ...
and on this one we have
if oc get -n $NAMESPACE secret ...

Reply:
oooo. you're right. Let me check this.